src/other/codemirror/CodeMirror.tsx

```diff
@@ -60,26 +77,49 @@ class CodeMirror extends React.Component<CodeMirrorProps, CodeMirrorState> {
 
     // eslint-disable-next-line class-methods-use-this
     getLanguageExtensions(language: string) {
-        if (LANGUAGES_MAP[language]) return LANGUAGES_MAP[language];
+        const baseExtensions = LANGUAGES_MAP[language] || LANGUAGES_MAP.fortran;
+
+        if (language === "exaxyz") {
+            return [...baseExtensions, checksStateField, linter(exaxyzLinter)];
+        }
```

Comment:
in case if other languages added this if/else will grow.
Better define `exaxyz` lang in `LANGUAGES_MAP` - this structure was created for that.